docs/devel/integration.rst

```diff
@@ -116,24 +118,24 @@ Introducing new strings
 +++++++++++++++++++++++
 
 You can add new strings in Weblate with :ref:`component-manage_units` turned
```

Comment:
Does having it on allow adding strings (correct now), or does that add strings btw.?

"When it is on"? Is it off by default? Manually? Not a very strong sentence.